packages/Cmfcmf-React-Fiber.package/CMFFiberTest.class/instance/testUseEffectWithEmptyDeps.st

```diff
@@ -0,0 +1,14 @@
+tests - useEffect
+testUseEffectWithEmptyDeps
+
+	| runs |
+	runs := OrderedCollection new.
+	self openComponentBlock: [
+		CMFReact
+			useEffect: [
+				runs add: #effect.
+				[runs add: #cleanup]]
+			dependencies: {}].
+	self assert: #(#effect) equals: runs asArray.
+	react reRenderAll.
+	self assert: #(#effect) equals: runs asArray
```

Comment:
This test makes me wonder whether we should cleanup effects when `react` is garbage collected.

Reply:
Interesting discussion. I believe for e.g. the Godot host this could lead to troubles, as the point where React is actually garbage collected Godot is most likely already closed. However, a shutdown event of some sort that the hosts could send could be actually quite beneficial (we already experience this with custom allocations in Godot where Godot now no longer cleanly exits because cleanup blocks don't release their references). I'll open an issue for further tracking.